core/pfe-core/controllers/internals-controller.ts

```diff
@@ -1,12 +1,85 @@
 import type { ReactiveController, ReactiveControllerHost } from 'lit';
 
-export class InternalsController implements ReactiveController {
+function isARIAMixinProp(key: string): key is keyof ARIAMixin {
+  return key === 'role' || key.startsWith('aria');
+}
+
+export class InternalsController implements ReactiveController, ARIAMixin {
+  declare role: ARIAMixin['role'];
```

Comment:
TS question.  Is there a reason we hare to `declare` these when we have already indicated that this class implements `ARIAMixin`

Reply:
`implements` just tells the compiler to complain if they aren't there